<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTexturesDialog</class>
 <widget class="QDialog" name="ViewTexturesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>791</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Texture Viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="searchLabel">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchTextureByNameInput">
           <property name="placeholderText">
            <string>Texture name...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="texturesTableView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addTextureButton">
           <property name="text">
            <string>New texture...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportTEXButton">
           <property name="text">
            <string>Export TEX</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="textureViewBox">
      <property name="title">
       <string>Texture:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="lodLabel">
             <property name="text">
              <string>LOD:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="mipLevelCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="formatLabelP">
             <property name="text">
              <string>Format:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="formatLabel">
             <property name="text">
              <string>Unkown</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="previewTabGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="texturePreview">
          <attribute name="title">
           <string>Texture</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="textureViewWidget">
             <property name="minimumSize">
              <size>
               <width>512</width>
               <height>512</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="replaceTextureButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>REPLACE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportTextureButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>EXPORT</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
